conn-ntg/README.md

```diff
@@ -0,0 +1,57 @@
+# Connect Native Trading Gateway (NTG)
```

Comment:
Please add information about the reconnectTimeout option with a default value 5000

Reply:
done